libs/blocks/hero-marquee/hero-marquee.js

```diff
@@ -175,7 +175,7 @@ export default async function init(el) {
     : null;
   if (assetUnknown) assetUnknown.classList.add('asset-unknown');
 
-  decorateBlockText(copy, ['xxl', 'm', 'l']); // heading, body, detail
+  decorateBlockText(copy, ['xxl', 'm', 'l'], 'allowDetailHeading'); // heading, body, detail
```

Comment:
Naming nit: the function parameter of `decorateBlockText ` is `type`, while `allowDetailHeading` sounds more like an option/configuration to me. Maybe `hasDetailHeding` or a verb-based name is better suited for a type.